### ISSUE TITLE
GDPR/legal requirements when scraping European citizen's data?

### ISSUE THREAD

**Issue by zacharylawrence** [Recidiviz/recidiviz-data#601]

**Comment by jessex** [comment]
I looked into this question some months ago but just refreshed myself. 

We do not have any GDPR requirements for European citizens processed in the American criminal justice system on American soil. In short, GDPR applies to data collected about individuals (not just European citizens) while they are in an EU state when the data is collected. It does _not_ apply to Europeans while they are abroad in a non-EU state when the data is collected. 

A relevant quote: "the personal data of an EU citizen residing in the US, for example, would be dealt with according to individual data protection laws within the US and would not be subject to GDPR compliance, whereas the personal data of a US citizen residing in the EU would be subject to GDPR regulations."

Where we _would_ require GDPR compliance is if we began gathering criminal justice data for European jurisdictions. That is a far off concern and not worth prioritizing now. But in the future it would be worth ensuring we are generally GDPR-friendly to afford us that opportunity.

If this seems like a reasonable enough answer for now, we can close this out. If you'd like, I'd be happy to put together documentation somewhere in the repository on this and other questions of legality and ethics.

Two sources of many: [1](https://www.compliancejunction.com/does-gdpr-apply-to-eu-citizens-in-the-united-states/), [2](https://www.hipaajournal.com/does-gdpr-apply-to-eu-citizens-living-in-the-us/).
